guides/web-apps/html-view-guidelines/README.md

```diff
@@ -181,13 +189,13 @@ To add `declarative-shadow-dom` to this HTML view, something like this can be do
 </template>
 ```
 
-Here, the elements are distributed in the way that the view will look when no blending is applied or when the app is running in standalone mode. 
+Here, the elements are distributed in the way that the view will look when no blending is applied or when the app is running in standalone mode.
 
 ### Guideline 5: Apply Styling to Avoid Conflicts and Allow Blending
 
 Regarding styling, there are two ways to make the application easier to visually integrate with other apps:
 
-1. Prefix the all class names with the name of the app. As outlined in [Avoiding CSS Conflicts](https://docs.starcounter.io/guides/blending/avoiding-css-conflicts/), the class should be prefixed with the name of the app to avoid CSS conflicts with classes from other apps.
+1. Prefix  all class names with the name of the app. As outlined in [Avoiding CSS Conflicts](https://docs.starcounter.io/guides/blending/avoiding-css-conflicts/), the class should be prefixed with the name of the app to avoid CSS conflicts with classes from other apps.
```

Comment:
There's an extra space after "Prefix".

I would also remove the redundant second sentence so that it becomes:
> Prefix all class names with the name of the app, as outlined in [Avoiding CSS Conflicts(https://docs.starcounter.io/guides/blending/avoiding-css-conflicts/).